pyproject.toml

```diff
@@ -60,3 +60,27 @@ include = [
   "/README.md",
   "/requirements.txt",
 ]
+
+[tool.ruff]
+target-version = "py37"
+extend-select = [
+  "I",
+]
+ignore = [
+  "E501", # from scripts/lint_backend.sh
+  "E722", # from scripts/lint_backend.sh
+  "E731", # from scripts/lint_backend.sh
+  "F403", # from scripts/lint_backend.sh
+  "F541", # from scripts/lint_backend.sh
+]
+
+[tool.ruff.per-file-ignores]
+"demo/*" = [
+  "E402", # Demos may have imports not at the top
+  "E741", # Demos may have ambiguous variable names
+  "F405", # Demos may use star imports
+  "I", # Don't care about import order
+]
+"gradio/__init__.py" = [
```

Comment:
We can get rid of this since you already added `__all__` right?

Reply:
Nope - that was a different module (`client/python/gradio_client/__init__.py`). 

I felt like adding about 80 `__all__` entries would've been a bigger change that can be done separately. 😁